data/net/minecraft/world/entity/ai/attributes/RangedAttribute.mapping

```diff
@@ -1,3 +1,19 @@
-CLASS net/minecraft/world/entity/ai/attributes/RangedAttribute
+CLASS net/minecraft/world/entity/ai/attributes/RangedAttribute net/minecraft/world/entity/ai/attributes/RangedAttribute
+	COMMENT Defines an entity ettribute that is limited to a range of values.
+	FIELD maxValue D
+		COMMENT The highest possible value for the attribute.
+	FIELD minValue D
+		COMMENT The lowest possible value for the attribute.
+	METHOD <init> (Ljava/lang/String;DDD)V
+		ARG 1 description
+		ARG 2 defaultValue
+		ARG 4 min
+		ARG 6 max
+	METHOD getMaxValue ()D
+		COMMENT Gets the highest possible value for the attribute.
+		COMMENT @return The highest possible value for the attribute. {@link maxValue}
+	METHOD getMinValue ()D
+		COMMENT Gets the lowest possible value for the attribute.
+		COMMENT @return The lowest possible value for the attribute. {@link minValue}
 	METHOD sanitizeValue (D)D
```

Comment:
You need a documentation line here at least.
Since the logic changes here for the sanitation.